### PR TITLE
Removing tourettes

### PR DESCRIPTION
**Pull request by Kaedwuff**
Removes the tourettes trauma from the game.

It's disruptive and obnoxious, and encourages spamming in the playerbase whenever it comes up.  It's also got very little to do with the actual disability it's referencing as it's written.

I would have tried rewriting it, but I can't actually seem to find any files that let me modify the behavior of the disability.  I assume they are serverside.  So I just removed the game's references to it so it won't show up.

### CODE REVIEW

**Comment by LordFowl** [comment]
> I assume they are serverside. So I just removed the game's references to it so it won't show up.
That is an incorrect assumption.

**Comment by Kaedwuff** [comment]
Where are they located, then?

**Comment by LordFowl** [comment]
Probably somewhere in speech code. A cursory string search is likely to pick it up.

**Comment by HeyBanditoz** [comment]
Port brainmed too pls tyty

File: html/changelogs/Changelog - Tourettes.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rsdel: "After PISS consideration, CUNT has removed tourettes SHIT the game."
```
Comment (skull132) [inline html/changelogs/Changelog - Tourettes.yml]: Please don't be stupid in changelogs.

**Comment by skull132** [comment]
@Kaedwuff that's not how code works. We can't easily define stuff like that on the server instance.

Look at code/modules/mob/living/carbon/human/life.dm line 205 and onwards, and code/modules/mob/living/carbon/human/say.dm line 236 and onwards to see where exactly tourettes does its thing. The former is the regular cursing, the latter is triggered on say.

**Comment by Kaedwuff** [comment]
I'm going to close this until I can make a Better Effort then. Thanks for the feedback!

**Pull request closed without merging**